lib/l10n/l10n.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+extension AppLocalizationsExtensions on BuildContext {
```

Comment:
should be `BuildContextExt`, we will use more extension functions from this context